spyder/plugins/editor/extensions/tests/test_docstring.py

```diff
@@ -0,0 +1,225 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+#
+"""Tests for close quotes."""
+# Third party imports
```

Comment:
Add a blank line here to cleanly separate the ``import``s from the header and docstring (it looks strange to have one line between import sections, and two after, but none before, and doesn't match our convention elsewhere).